lib/apex-class-cleanup.js

```diff
@@ -27,6 +28,7 @@ let login = function(options) {
     let connection = new jsforce.Connection(config);
```

Comment:
add after the let on it's own line

`connection.metadata.pollTimeout = options.pollTimeout || FallbackPollTimeout;`

Reply:
disregard! I was wrong!